compute/client_library/snippets/disks/resize_disk.py

```diff
@@ -0,0 +1,122 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+
+
+# This file is automatically generated. Please do not modify it directly.
+# Find the relevant recipe file in the samples/recipes or samples/ingredients
+# directory and apply your changes there.
+
+
+# [START compute_regional_disk_resize]
+# [START compute_disk_resize]
```

Comment:
Why do we have the same sample for two region tags?

Reply:
Because I tend to write more universal samples, while other languages might need to have  separate methods for each. For example with static typing, it wouldn't be possible to have `disk_client` variable holding `compute_v1.RegionDisksClient()` or `compute_v1.DisksClient()` depending on context.

With this sample, the code is not as clean as I hoped, due to different names of used fields, but sometimes this works really nicely.

Reply:
Makes sense. Thanks for clarifying.